sample/src/main/res/values/strings.xml

```diff
@@ -15,7 +15,11 @@
     <string name="transaction">Transaction</string>
     <string name="send_transaction">Send Transaction</string>
     <string name="get_kin">Get KIN</string>
-    <string name="create_account">Create Account</string>
+    <string name="delete_wallet">Delete Wallet</string>
+    <string name="delete_wallet_warning">Are you sure you want to delete your wallet?\n\nDelete without backed up will cause permanent funds lost</string>
```

Comment:
Are you sure you want to delete your wallet?\n\nWarning: deleting a wallet when it hasn't been backed up will cause funds to be lost. Use Export Key Store to backup your wallet.

Reply:
this is too long for this dialog
need to give the shortest message that will give this info.
no word of export in ios - no need to add it here impo

Reply:
agree with @shayb